recipes-support/swupdate/swupdate/defconfig

```diff
@@ -27,3 +27,4 @@ CONFIG_PARSERROOT=""
 CONFIG_RAW=y
 CONFIG_ARCHIVE=y
 CONFIG_BOOTLOADERHANDLER=y
+# CONFIG_MTD is not set 
```

Comment:
This is a comment, right? Is it needed?

Reply:
Just for someone running into this:

* https://www.yoctoproject.org/docs/2.5/kernel-dev/kernel-dev.html#using-menuconfig
* https://unix.stackexchange.com/questions/402650/commented-sections-in-kernel-config/426500 (which seems incorrect in regards to Yocto)
* https://www.openembedded.org/pipermail/openembedded-core/2016-May/239350.html